### DISCUSSION TITLE
Lists

### DISCUSSION THREAD

**Discussion by ricmazz** [community/community#41060] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Very interesting the new feature 'Lists' in Stars, I think now it will be simpler to classify the contents that I starred. This more categorized way will make it easier to find or target starred content, in my opinion.

Has anyone else tried the new feature as well? What did you think?

**Comment by jfullstackdev** [comment]
that's quite a jumbled translation 

![image](https://user-images.githubusercontent.com/47092464/206381290-e6df770b-b054-4e08-a1e3-ccea6ce481b8.png)

can you rearrange it, so we can understand too

**Reply by ricmazz** [reply]
Thanks for your recommendation, I edited my comment to English and rewrote it in a simpler way.

**Reply by LHS7197** [reply]
Did you post your updated comments
